src/test/java/seedu/clinic/logic/commands/EditCommandTest.java

```diff
@@ -119,10 +120,12 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
 
         WarehouseBuilder warehouseInList = new WarehouseBuilder(lastWarehouse);
         Warehouse editedWarehouse = warehouseInList.withName(VALID_NAME_BOB)
-                .withPhone(VALID_PHONE_BOB).build();
+                .withPhone(VALID_PHONE_BOB)
+                .withProducts(Map.of(VALID_WAREHOUSE_PRODUCT_NAME_A, 50)).build();
 
         EditWarehouseDescriptor descriptorWarehouse = new EditWarehouseDescriptorBuilder()
-                .withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB).build();
+                .withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
+                .withProducts(Map.of(VALID_WAREHOUSE_PRODUCT_NAME_A, 50)).build();
```

Comment:
I think you can use one of the valid quantities found in the util instead of putting a number here

Reply:
I've made the change, thanks!